pkg/util/mock-exec-run.go

```diff
@@ -0,0 +1,44 @@
+package util
```

Comment:
can we have following packaging structure & file name:
`pkg/exec/mock/v1alpha1/exec.go`

Reply:
sounds good

Reply:
Sounds good